### PR TITLE
[risk=low][RW-7921] Added Data Access column to user admin table

### PR DESCRIPTION
**Pull request by evrii**
Description:
image

Added Data Access to user admin table. Access is shown as a combination of badges or "N/A"

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [x] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: ui/src/app/pages/admin/user/admin-user-table.tsx [hunk 1]
```diff
@@ -152,6 +154,18 @@ export const AdminUserTable = withUserProfile()(
...
     }
 
+    dataAccessContents(user: AdminTableUser): JSX.Element {
+      return (
+        <FlexRow style={{ justifyContent: 'space-evenly' }}>
+          {user.accessTierShortNames.length > 0
+            ? user.accessTierShortNames.map((accessTierShortName) => (
+                <TierBadge {...{ accessTierShortName }} />
+              ))
+            : 'N/A'}
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/user/admin-user-table.tsx:164]: This logic is almost right ... but I see that the story doesn't call this out explicitly and it probably should have.

"N/A" is for the case when the user is **disabled** and this means that we don't calculate their appropriate access tiers.

We know when the user is Enabled-but-no-tiers, so we can show something different here.  Maybe "no access" like we have in AdminUserProfile.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
+*.nix
```
Comment (jmthibault79) [inline .gitignore:1]: what is this?

**Review by jmthibault79 — APPROVED** [review]
looks great, tested locally

File: .gitignore (hunk 2 above)
Comment (evrii) [reply .gitignore]: I had some local scripts that I was keeping in my local repo and trying not to commit. I have them in a folder outside of the repo, so this is no longer necessary.

**Pull request merged**